### DISCUSSION TITLE
I can't seem to import enable3d as a module in my site

### DISCUSSION THREAD

**Discussion by kvasserman** [enable3d/enable3d#391] [Help / Questions] [answered]
I've downloaded enable3d.framework.0.25.4.min.js and imported it in my HTML, like so:

`        <script type="importmap">
            {
              "imports": {
                "enable3d": "./js/enable3d.framework.0.25.4.min.js"
              }
            }
        </script>`

GET request for the js file works fine, the browser downloads the enable3d source successfully.

In my code module js code I try to import anything from it, but keep getting an error. For example, this

`import { Project } from 'enable3d';`

results in this error:

`The requested module 'http://localhost:5000/js/enable3d.framework.0.25.4.min.js' doesn't provide an export named: 'Project'`

I feel like I'm missing something stupid. Any help would be greatly appreciated.

Thank you in advance.

**Comment by Aaryan-Umar** [comment]
I'm having the same problem, I have even tried to copy a standalone example but it is still not working

**Reply by yandeu** [reply, marked as answer]
Module support will be added in version 0.26.0.

Some examples (like this one https://enable3d.io/examples/native-three-with-physics.html) have already been updated.
